intel-retail/retail-use-cases
# feat: add tensorflow keras classification pipeline

## PR Checklist



## What are you changing?
Added tensorflow keras classification pipeline with performance evaluation feature

## Issue this PR will close

close: #**52**

## Anything the reviewer should know when reviewing this PR?
Model is not detecting all objects, experimenting with other models. Will replace if I find any better that is available in keras.

## Test Instructions if applicable

- Make sure input source video is added to performance-tools/sample-media folder and update path in [tensorflow-keras-env.txt](https://github.com/intel-retail/retail-use-cases/compare/main...NeethuES-intel:retail-use-cases:tensorflow-integration?expand=1#diff-67158b975325d077d52e85c421b67051d3471dc0cbb77d22ce6765f4711a00ac)
- make build_tensorflow_keras
- make run_tensorflow_keras_classification
- run 2 pipelines with PIPELINE_COUNT=2 make run_tensorflow_keras_classification
- make down_tensorflow_keras_classification
- results of the detection (only object detected) is published to mqtt broker (topic - tensorflow_keras_classification_results) and logged into results.txt, fps in pipelines.txt

## If the there are associated PRs in other repositories, please link them here (i.e. intel-retail/retail-use-cases )
